client/package.json

```diff
@@ -4,6 +4,7 @@
     "author": "Polymer Authors",
     "license": "MIT",
     "version": "0.1.1",
+		"icon": "./polymer.png",
```

Comment:
What a weird mistake. I don't know.. maybe some autoformat thing happened to the file? Or maybe I was really in a rush?

Fixed!